person.rb

```diff
@@ -0,0 +1,17 @@
+class Person
+  attr_accessor :id, name, age
+
```

Comment:
- [ ] You still don't have the constructor as requested by the previous reviewer. You should have something like this at the top of the class:

  ```
  def initialize(age, name = 'Unknown', parent_permission: true)
    @id = rand(1...1000)
    @age = age
    @name = name
    @parent_permission = parent_permission
  end
  ```